src/main/kotlin/org/malachite/estella/process/domain/RecruitmentProcessDto.kt

```diff
@@ -30,15 +36,37 @@ fun RecruitmentProcess.toRecruitmentProcessDto() = RecruitmentProcessDto(
 
 data class TaskDto(
     val id: Int?,
-    val tests: String,
-    val description: String,
+    val testsBase64: String,
+    val descriptionFileName: String,
+    val descriptionBase64: String,
     val timeLimit: Int,
     val deadline: Timestamp
 )
+
+fun TaskDto.toTask(): Task {
+    print(String(Base64.getDecoder().decode(this.descriptionBase64)))
+    return Task(
+        this.id,
+        SerialBlob(Base64.getDecoder().decode(this.testsBase64)),
+        SerialClob(String(Base64.getDecoder().decode(this.descriptionBase64)).toCharArray()),
```

Comment:
Czemu robisz String(...), zamiast .toString()?

Reply:
Ogólnie toString nie robiło tego samego. Jesli używasz używasz toString na ByteArray to wychodzi Ci jakieś coś takiego: [B342bzue. Więc to raczej nie było to. Zgaduję, że konstruktor Stringa potrafi sobie parsnąć to, co jest potrzebne, żeby to jakiś sens miało. W zadaniu z rabbita chyba też Stringa wołaliśmy, a nie toString.
Plus wydaje mi się, że toString to jest odpowiednik pythonowego repr. To po prostu wypisuje Ci, co to jest, nie stringa zbudowanego z tego.

Reply:
Aha ok czyli ByteArray nie ma zaimplementowanej właśnej wersji toString, tylko korzystam z tej domyślnej Javowej implementacji wskazujacej na adres, ale bieda XD